pubspec.yaml

```diff
@@ -4,21 +4,22 @@ version: 3.0.0
 homepage: https://docs.atsign.com/
 
 environment:
-  sdk: '>=2.16.1 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
```

Comment:
This will need #136 and #139 to be merged so that Dart 3 is in place for the binaries and Docker images. It might be worth merging their branches into this so it all gets done at once.

Reply:
I've rolled those two PRs into this one